services/add-media-ids/now.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": 1,
+  "alias": "frontity-add-media-ids",
```

Comment:
Don't you need to add `.now.sh` to the alias?

Reply:
Nopes

Reply:
Oki doki